frontend/src/containers/GroupApplication/index.tsx

```diff
@@ -1,22 +1,7 @@
-import React, { useEffect } from "react";
-import { getApplictionTextDrafts } from "src/utils/draftHelper";
+import React from "react";
 import Application, { ApplicationProps } from "./Application";
 
 const GroupApplication = (props: ApplicationProps) => {
-  useEffect(() => {
-    initializeValue();
-  }, []);
-
-  const initializeValue = () => {
-    const groupName = props.group.name.toLowerCase();
-
-    const restoredApplicationText = getApplictionTextDrafts();
-
-    if (restoredApplicationText != null && restoredApplicationText[groupName]) {
-      props.form.setFieldValue(groupName, restoredApplicationText[groupName]);
-    }
-  };
-
```

Comment:
This code has the same functionality as what I modified in `Applicationform/index.tsx` line 111. No need to do it twice.